binding/SkiaSharp/SKTextBlob.cs

```diff
@@ -403,10 +394,16 @@ public SKRunBuffer AllocateRun (SKFont font, int count, float x, float y, SKRect
 			else
 				SkiaApi.sk_textblob_builder_alloc_run (Handle, font.Handle, count, x, y, null, &runbuffer);
 
-			return new SKRunBuffer (runbuffer, count);
+			buffer = new SKRunBuffer<float> (runbuffer, count, 0, 0);
 		}
 
 		public SKTextRunBuffer AllocateTextRun (SKFont font, int count, float x, float y, int textByteCount, SKRect? bounds = null)
+		{
+			AllocateTextRun (font, count, x, y, textByteCount, bounds, out var buffer);
+			return new SKTextRunBuffer (buffer.buffer, count, textByteCount);
+		}
+
+		public void AllocateTextRun (SKFont font, int count, float x, float y, int textByteCount, SKRect? bounds, out SKRunBuffer<float> buffer)
```

Comment:
This new API may be annoying to use. Should we just rename the struct-returning ones and return directly?

```cs
public SKRunBuffer<float> AllocateTextRunSpan (SKFont font, int count, float x, float y, int textByteCount, SKRect? bounds)
```

Reply:
Going with this idea for now until I get feedback. The params are now the same, the method name now is the controlling factor.